onnxruntime/core/dlpack/dlpack_converter.cc

```diff
@@ -208,7 +208,7 @@ DLManagedTensor* OrtValueToDlpack(OrtValue& ort_value) {
   ort_dlmanaged_tensor->tensor.manager_ctx = ort_dlmanaged_tensor;
   ort_dlmanaged_tensor->tensor.deleter = &DlpackDeleter;
   ort_dlmanaged_tensor->tensor.dl_tensor.data = (tensor.MutableDataRaw());
-  ort_dlmanaged_tensor->tensor.dl_tensor.ctx = GetDlpackContext(ort_value, tensor.Location().device.Id());
+  ort_dlmanaged_tensor->tensor.dl_tensor.device = GetDlpackDevice(ort_value, tensor.Location().device.Id());
```

Comment:
is this change fine to support older version of PyTorch, for example 1.9?

Reply:
The new version of DLPack just renamed some variables, the data structure is backward compatible. Since we don't link LibTorch, PyTorch and ORT can use different of DLPack to build, they are independent. The changes are tested both on PyTorch 1.9.1 and 1.10.0. The pipeline check here are also use 1.9.1, they work well.

Reply:
awesome, thanks :)